mug-compose.yml

```
@@ -71,7 +71,7 @@ services:
   web-ui:
     command: ["uwsgi /opt/invenio/var/instance/uwsgi_ui.ini"]
     pull_policy: always
-    image: ghcr.io/sharedrdm/instance:main-mug
+    image: ghcr.io/sharedrdm/instance:${RELEASE_TAG}
     restart: "unless-stopped"
     env_file: .env.temp
```

```suggestion
    env_file: .env
```
and add all the content of `.env.temp` to `.env` and have only one env file.